src/pymovements/base.py

```diff
@@ -29,6 +28,8 @@ class Screen:
         Screen height in centimeters
     distance_cm : float
         Eye-to-screen distance in centimeters
+    origin : str
+        Specifies the origin location of pixel coordinates.
```

Comment:
Specifies the origin location in pixel coordinates.

Reply:
I see the issue in my formulation.. but I would interpret the origin location IN pixel coordinates to be `(0, 0)`, because technically, this is the origin specified in pixel coordinates.

What do you think about that?

``` 
Specifies the screen location of the origin in the pixel coordinate system.
```

Reply:
sounds good to me

Reply:
After some googling I think it's the origin of a space/coordinate system instead of in.
So I used that.